### ISSUE TITLE
Can import data to a disconnected redis server

### ISSUE THREAD

**Issue by codejamtc** [topcoderinc/TC-redis-manager#308]
![image](https://user-images.githubusercontent.com/42398485/51068513-eded7600-15e4-11e9-8672-641a488a6fc5.png)

1. Deploy the App and run the application
2. Create 5 redis servers with different names
3. Disconnect one redis servers
4. Click import and select the disconnected server from the drop down
5. Select Flush DB and click import

**Actual:** Can import data to a disconnected redis server

**Expected:** Should not allow data import to a disconnected one. Should remove the disconnected redis servers from the drop down

**Browser(s):** Firefox 64.0.2 (Windows)

**Comment by standlove** [comment]
Yes, please remove the disconnected server from the export dialog dropdown list as well.
